flixel/addons/tile/FlxTilemapExt.hx

```diff
@@ -495,15 +502,49 @@ class FlxTilemapExt extends FlxTilemap
 	/**
 	 * Sets the tiles that are treated as "clouds" or blocks that are only solid from the top.
 	 * 
-	 * @param 	Clouds	An array containing the numbers of the tiles to be treated as clouds.
+	 * @param 	Clouds		An array containing the numbers of the tiles to be treated as clouds.
+	 * @param 	Conditional	Set cloud tiles to be treated as conditional.
 	 */
-	public function setClouds(?Clouds:Array<Int>):Void
+	public function setClouds(?Clouds:Array<Int>, Conditional:Bool = false):Void
 	{
 		if (Clouds != null)
 		{
 			for (i in 0...(Clouds.length))
 			{
-				setTileProperties(Clouds[i], FlxObject.CEILING);			
+				setTileProperties(Clouds[i], FlxObject.CEILING);
+				if (Conditional)
+				{
+					cast(_tileObjects[Clouds[i]], FlxTile).callbackFunction = solveCollisionCloud;
+				}
+			}
+		}
+	}
+	
+	/**
+	 * Solves collision against a cloud tile
+	 * 
+	 * @param 	Slope 	The slope to check against
+	 * @param 	Object 	The object that collides with the slope
+	 */
+	private function solveCollisionCloud(Slope:FlxObject, Object:FlxObject):Void
```

Comment:
This doesn't really "solve" any collisions, just set the `allowCollisions` property.